### PR TITLE
Fix footer

### PR DESCRIPTION
**Pull request by mjrb**
plz accept the other one that does nothing, i think it'll fix my fork

### CODE REVIEW

File: events.html [hunk 1]
```diff
@@ -60,5 +60,6 @@ <h4>To find out about more events outside this month, checkout the offical USACS
         <a href="https://usacs.slack.com/"><img src="assets/img/slack-mark.png"></a>
       </div>
     </footer>
+    <script src="assets/js/footer.js"></script>
```
Comment (hemangandhi) [inline events.html]: Shouldn't this be in a template somewhere?

File: assets/js/footer.js [hunk 2]
```diff
@@ -0,0 +1,9 @@
+function FIX_THIS_S___(){
```
Comment (hemangandhi) [inline assets/js/footer.js]: May be `fixFooter`?

File: events.html (hunk 1 above)
Comment (mjrb) [reply events.html]: It is in the template, that why it got compiled in

File: assets/js/footer.js (hunk 2 above)
Comment (mjrb) [reply assets/js/footer.js]: Oh that was a joke I forgot to change it to something clean. It's a good thing I censored myself in anticipation of not remembering to change it

File: events.html (hunk 1 above)
Comment (hemangandhi) [reply events.html]: Oh yeah... I'll just have to ctrl+f "mustache" for all the template things.

File: assets/js/footer.js (hunk 2 above)
Comment (hemangandhi) [reply assets/js/footer.js]: I'll merge once you make this a better name.

**Comment by hemangandhi** [comment]
I think merging this might fix your fork. What exactly is wrong with it?

**Pull request closed without merging**